### PR TITLE
[JAVA] Relax the CMAP documentation requirement for durations in events

### PR DESCRIPTION
**Pull request by rachel-mack**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-35270>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-35270/whats-new/#std-label-version-5.0>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -54,6 +54,10 @@ The 5.0 driver release introduces the following features:
   name, or it defaults to ``MULTIPLE`` if you specify more than one
   host.
 
+- The ``ConnectionCheckedOutEvent.getElapsedTime`` method now includes the time
+  taken to create the connection. That is, the "timer" starts before the
+  connection is confirmed, rather than after.
+
```
Comment (mongoKart) [inline source/whats-new.txt:60]: I think more technical detail would be helpful here. I'd probably just paste in the Jira notes and maybe edit them slightly.

Also, I think the ticket describes more changes than I see here; is there another ticket for those?

**Review by mongoKart — CHANGES_REQUESTED** [review]

**Comment by rachel-mack** [comment]
Note: Vale error is for use of "Fail" in the event name.

File: source/whats-new.txt [hunk 2]
```diff
@@ -59,6 +59,18 @@ The 5.0 driver release introduces the following features:
   ``isNumber()`` and ``asNumber()`` methods match the Java responses for
   equivalent ``Decimal128`` values.
 
+- The ``getElapsedTime`` method on ``com.mongodb.event.ConnectionReadyEvent``
+  includes the time taken to deliver the ``ConnectionPoolListener``. That is,
```
Comment (mongoKart) [inline source/whats-new.txt:63]: I: per ticket, i think this should be:

```suggestion
  includes the time taken to deliver the ``ConnectionCreatedEvent``. That is,
```

File: source/whats-new.txt [hunk 3]
```diff
@@ -59,6 +59,18 @@ The 5.0 driver release introduces the following features:
   ``isNumber()`` and ``asNumber()`` methods match the Java responses for
   equivalent ``Decimal128`` values.
 
+- The ``getElapsedTime`` method on ``com.mongodb.event.ConnectionReadyEvent``
```
Comment (mongoKart) [inline source/whats-new.txt:62]: I: In all of our docs that I've seen, we suffix method names with parentheses

```suggestion
- The ``getElapsedTime()`` method on ``com.mongodb.event.ConnectionReadyEvent``
```

File: source/whats-new.txt [hunk 4]
```diff
@@ -59,6 +59,18 @@ The 5.0 driver release introduces the following features:
...
+  includes the time taken to deliver the ``ConnectionPoolListener``. That is,
+  the time returned includes the duration of
+  ``com.mongodb.event.ConnectionPoolListener.connectionCreated``.
```
Comment (mongoKart) [inline source/whats-new.txt:65]:
```suggestion
  the time returned includes the duration of the
  ``com.mongodb.event.ConnectionPoolListener.connectionCreated()`` method.
```

File: source/whats-new.txt [hunk 5]
```diff
@@ -59,6 +59,18 @@ The 5.0 driver release introduces the following features:
...
+  
+  The ``getElapsedTime`` methods on
```
Comment (mongoKart) [inline source/whats-new.txt:67]:
```suggestion
  The ``getElapsedTime()`` methods on
```

File: source/whats-new.txt [hunk 6]
```diff
@@ -59,6 +59,18 @@ The 5.0 driver release introduces the following features:
...
+  ``com.mongodb.event.ConnectionCheckedOutFailedEvent`` and
+  ``com.mongodb.event.ConnectionCheckedOutEvent`` include the time taken to
+  deliver the ``com.mongodb.event.ConnectionCheckOutStartedEvent``. That is, the
+  time returned includes the duration of
+  ``com.mongodb.eventConnectionPoolListener.connectionCheckOutStarted``.
```
Comment (mongoKart) [inline source/whats-new.txt:72]:
```suggestion
  time returned includes the duration of the
  ``com.mongodb.eventConnectionPoolListener.connectionCheckOutStarted()`` method.
```

**Review by mongoKart — CHANGES_REQUESTED** [review]
just a couple quick changes!

**Review by mongoKart — APPROVED** [review]
LGTM

**Pull request merged**